exporter.json

```diff
@@ -317,6 +317,29 @@
           }
         ]
       },
+      {
+        "key": "io.supernova.documentation-main.component-status",
+        "category": "Components",
+        "title": "R7 Component Status",
```

Comment:
I wouldn't prepend the name with anything, because then it would have to be called as /r7... with the slash commands in editor, which isn't super intuitive. Let's go with `Component Status (R7)` or `Component Status (R7 Custom)` 👍

Reply:
i tested (R7 custom) and it pushes it onto a new line which doesn't look good at all, so probably better to go with (R7)